example/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "react-csjs-example",
+  "version": "1.0.0",
+  "description": "Example app showing simple react-csjs use",
+  "scripts": {
+    "dev": "kyt dev",
+    "test": "kyt test"
+  },
+  "author": "Jeremy Gayed (http://github.com/tizmagik)",
+  "license": "MIT",
+  "dependencies": {
```

Comment:
kyt docs recommend adding it as a dependency to your project, which we should probably do (I didn't have it installed, so `run dev` failed immediately).